app/services/github_checks_verifier.rb

```diff
@@ -45,16 +46,20 @@ def filters_present?
     check_name.present? || check_regexp.present?
   end
 
+  def check_conclusion_allowed(check)
+    allowed_conclusions.include? check.conclusion
+  end
+
   def fail_if_requested_check_never_run(check_name, all_checks)
     return unless check_name.present? && all_checks.blank?
 
     raise StandardError, "The requested check was never run against this ref, exiting..."
   end
 
-  def fail_unless_all_success(checks)
-    return if checks.all? { |check| check.conclusion == "success" }
+  def fail_unless_all_conclusions_allowed(checks)
+    return if checks.all? { |check| check_conclusion_allowed(check) }
 
-    raise StandardError, "One or more checks were not successful, exiting..."
+    raise StandardError, "The conlclusion of one or more checks were not allowed. Allowed conclusions are: [#{allowed_conclusions}]. This can be configured with the 'allowed_conclusions' param."
```

Comment:
just a small typo (my fault when I wrote it). The param name for the end-user is `allowed-conclusions` ('-' instead of '_')

Reply:
And another typo: `conlclusion`